### DISCUSSION TITLE
Elasticsearch

### DISCUSSION THREAD

**Discussion by gcherem** [prisma/prisma#2997] [Q&A] [answered]
Hi folks!

First, I want to thank you for this great project!

I would like to know if is there plans for supporting elasticsearch.

Keep up the good work!

**Comment by ryands17** [comment, marked as answer]
Hey @gcherem :wave: 
Currently there are not plans, but could you open a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into this further :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
